proto/prom/find.go

```diff
@@ -104,40 +105,87 @@ func (p *QL) FindLabelsValues(ctx echo.Context) error {
 
 	token := core.RetrieveToken(r)
 	if len(token) == 0 {
-		respondWithError(w, errors.New("Not authorized, please provide a READ token"), http.StatusForbidden)
+		respondWithError(w, errors.New("not authorized, please provide a READ token"), http.StatusForbidden)
 		return nil
 	}
 
 	labelValue := ctx.Param("label")
-
 	if len(labelValue) == 0 {
-		log.WithFields(log.Fields{}).Error("missing label")
-		respondWithError(w, errors.New("Unprocessable Entity: label"), http.StatusBadRequest)
+		log.Error("missing label")
+		respondWithError(w, errors.New("unprocessable Entity: label"), http.StatusBadRequest)
+		return nil
+	}
+
+	// Parse form data
+	if err := r.ParseForm(); err != nil {
+		respondWithError(w, errors.New("failed to parse form data"), http.StatusBadRequest)
+		return nil
+	}
+
+	// Get matchers and process them
+	matchers := r.Form["match[]"]
+	if len(matchers) == 0 {
+		// Grafana will try to get all class name when arriving explore page
```

Comment:
I'm not sure we should return nothing if no filters, most of the users does not have performance issues, open to discuss

Reply:
The problem here is that by default Grafana will try to fetch all series, we might not want this to prevent overwhelming of erlenmeyer. If we do not implement a by default `200`, Grafana will get an error and be printed to the user, showing a misleading error. 

Even if series are already fetched, when we click the input for serie search, it will try again to search, I belive this behaviour is acceptable in this specific case.